.github/workflows/module_release.yml

```diff
@@ -24,7 +29,22 @@ jobs:
       - name: "Get version"
         id: "get_version"
         run: |
-          echo "version=$(jq --raw-output .version metadata.json)" >> $GITHUB_OUTPUT
+          ruby <<-EOF >> $GITHUB_OUTPUT
+            require "json"
+            version = JSON.parse(File.read("metadata.json"))["version"]
+
+            # from https://github.com/voxpupuli/metadata-json-lint/blob/b5e68049c6be58aa63263357bb0dcad8635a6829/lib/metadata-json-lint/schema.rb#L141-L150
```

Comment:
If you're OK using a library (imo 👍) then this bit can all be replaced with

`version = SemanticPuppet::Version.parse(JSON.parse(File.read("metadata.json"))["version"])`

Reply:
The trouble there is that adds a gem dep to this action.

Reply:
Although using a cleaner method would introduce a new gem dependency, we should stick with @donoghuc 's approach to avoid adding any additional dependencies.